proposals/nnnn-variadic-types.md

```diff
@@ -0,0 +1,239 @@
+# Allow Generic Types to Abstract Over Packs
+
+* Proposal: [SE-NNNN](nnnn-variadic-types.md)
+* Authors: [Slava Pestov](https://github.com/slavapestov), [Holly Borla](https://github.com/hborla)
+* Review Manager: TBD
+* Upcoming Feature Flag: `VariadicGenerics`
+* Previous Proposal: [SE-0393](0393-parameter-packs.md)
```

Comment:
Just checking, does the same line from SE-0393 apply here, i.e.:
> * Implementation: On main gated behind the frontend flag `-enable-experimental-feature VariadicGenerics`

?

Reply:
Yeah, I think we should replace 

> Upcoming Feature Flag: `VariadicGenerics`

with

> Implementation: On `main` and `release/5.9` gated behind the frontend flag `-enable-experimental-feature VariadicGenerics`

Because `VariadicGenerics` is an experimental feature, not an upcoming one.